include/constants/item_effects.h

```diff
@@ -2,18 +2,20 @@
 #define GUARD_CONSTANTS_ITEM_EFFECTS_H
 
 // field 0 masks
-#define ITEM0_X_ATTACK          0x0F
 #define ITEM0_DIRE_HIT          0x30 // Works the same way as the move Focus Energy.
 #define ITEM0_SACRED_ASH        0x40
 #define ITEM0_INFATUATION       0x80
 
 // field 1 masks
-#define ITEM1_X_SPEED           0x0F
-#define ITEM1_X_DEFEND          0xF0
+#define ITEM1_X_ATTACK          0x1
+#define ITEM1_X_DEFEND          0x2
+#define ITEM1_X_SPEED           0x4
+#define ITEM1_X_ACCURACY        0x8
```

Comment:
Is there a reason that Accuracy comes before Sp. Atk and Def while other orders show Accuracy being last?

Reply:
It's the order Game Freak followed for the actual items in the Gen. 4 games and those after it, so when Ketsuban brought it up in the original issue, I just decided to stick to that because why not :eyes:
https://bulbapedia.bulbagarden.net/wiki/List_of_items_by_index_number_(Generation_IV)
https://bulbapedia.bulbagarden.net/wiki/List_of_items_by_index_number_(Generation_V)
https://bulbapedia.bulbagarden.net/wiki/List_of_items_by_index_number_(Generation_VI)
https://bulbapedia.bulbagarden.net/wiki/List_of_items_by_index_number_(Generation_VII)
https://bulbapedia.bulbagarden.net/wiki/List_of_items_by_index_number_(Generation_VIII)